packages/cli/mock/src/runMockServer.ts

```diff
@@ -0,0 +1,208 @@
+import { assertNever } from "@fern-api/core-utils";
+import { generateIntermediateRepresentation } from "@fern-api/ir-generator";
+import { TaskContext } from "@fern-api/task-context";
+import { FernWorkspace } from "@fern-api/workspace-loader";
+import {
+    ExampleEndpointCall,
+    ExamplePathParameter,
+    ExampleTypeReference,
+    HttpEndpoint,
+    NameAndWireValue
+} from "@fern-fern/ir-sdk/api";
+import express, { Request, Response } from "express";
+import { ParamsDictionary, Query } from "express-serve-static-core";
+import getPort from "get-port";
+import { IncomingHttpHeaders } from "http";
+import { isEqual } from "lodash-es";
+import qs from "qs";
+import urlJoin from "url-join";
+
+type RequestHandler = (req: Request, res: Response) => void;
+
+export async function runMockServer({
```

Comment:
today examples don't include auth / global headers -- don't have to solve in this PR but we're going to need to figure that out in order for the sdk testing to work

Reply:
Good call, I'll add a TODO in-line for now (along with a mention to some of the other features we're eventually need to support like streaming, file upload, etc).

Reply:
Created a new issue [here](https://github.com/fern-api/fern/issues/2620).